x/mint/keeper/abci.go

```diff
@@ -18,12 +19,19 @@ func (k Keeper) BeginBlocker(ctx context.Context, mintFn types.MintFn) error {
 		return err
 	}
 
+	oldBz := k.cdc.MustMarshal(&minter)
```

Comment:
is there anyway we can get the raw bytes? marshaling right after unmarshaling seems like lots of overhead

Reply:
What do you mean with raw bytes? Like using reflection?